demos/test-app/src/auth.ts

```diff
@@ -0,0 +1,123 @@
+import type { SignIdentity, Signature } from "@dfinity/agent";
+import {
+  Delegation,
+  DelegationChain,
+  DelegationIdentity,
+} from "@dfinity/identity";
+import { Principal } from "@dfinity/principal";
+
+// The type of response from II as per the spec
+interface AuthResponseSuccess {
```

Comment:
You don't have that in a did files somewhere to avoid duplication and for to improve maintability?

Reply:
not in a did file but I have the type defined in II somewhere. Next I'm gonna extract it like I did for the VC types